charts/kong/UPGRADE.md

```diff
@@ -45,6 +45,33 @@ clear it.
 
 ## 1.4.0
 
+### Changes to default Postgres permissions
+
+The [Postgres sub-chart](https://github.com/bitnami/charts/blame/master/bitnami/postgresql/)
```

Comment:
This link should not have `blame`. The correct link is https://github.com/bitnami/charts/tree/master/bitnami/postgresql.